salt/modules/win_event_viewer.py

```diff
@@ -0,0 +1,417 @@
+# -*- coding: utf-8 -*-
+'''
+:codeauthor: Charles McMarrow <cmcmarrow@saltstack.com>
+gives access to Windows event log
+'''
+
+from __future__ import absolute_import
```

Comment:
Do we not need `print_function` and `unicode_literals` here anymore since we're migrating to Py3 on this branch?

Reply:
Pylint will fail if I don't have that